lib/fog/ovirt/models/compute/server.rb

```diff
@@ -44,7 +44,7 @@ def locked?
         end
 
         def stopped?
-          !!(status =~ /down/i)
+          !!(status =~ /^down$/i)
```

Comment:
not significant, but doesn't `status.downcase == 'down'` have the same effect while being faster and easier to read?